src/server/game/Entities/Pet/Pet.cpp

```diff
@@ -795,6 +795,11 @@ bool Pet::CreateBaseAtTamed(CreatureTemplate const* cinfo, Map* map)
     return true;
 }
 
+static float interpolate_base_damage (float level, float max_level, float min_level, float max_damage, float min_damage, float power) {
```

Comment:
Codestyle. You probably also want to put this on lookup tables for each level (pow is sloooow)